test.md

```diff
@@ -0,0 +1,52 @@
+
+---
+title: Event 1
```

Comment:
Just as a reminder: probably something to delete before you check in?

Or would `draft: true` in the front matter exclude this from most builds?

Reply:
Oh snap, didn't realize this made it in... Removing this..